aspnetcore/blazor/host-and-deploy/server.md

```diff
@@ -62,91 +62,103 @@ Recommendations for global deployments to geographical data centers:
+The service isn't required for Blazor apps hosted in Azure App Service but can be helpful in other hosting environments:
 
-The app must support *session affinity*, also called *sticky sessions*, where clients are redirected back to the same server. To support session affinity, an application setting in the Azure portal is automatically configured (**Configuration** > **Application settings**: **Name**: `Azure__SignalR__ServerStickyMode`, **Value**: `Required`). Therefore, you don't need to manually configure the app for session affinity.
+* To facilitate connection scale out.
+* Handle global distribution.
 
-If you prefer not to use the app setting in Azure (and delete it in the Azure portal), two other approaches for configuring session affinity are (**use either approach, not both**):
+:::moniker range=">= aspnetcore-8.0"
 
-* Set the option in `Program.cs`:
+<!-- UPDATE 9.0 Update section to only cross-link stateful
+                reconnect guidance after the feature is
+                supported with Azure SignalR Service. -->
 
-  ```csharp
-  builder.Services.AddSignalR().AddAzureSignalR(options =>
-  {
-      options.ServerStickyMode = 
-          Microsoft.Azure.SignalR.ServerStickyMode.Required;
-  });
-  ```
+> [!NOTE]
+> [Stateful reconnect](xref:signalr/configuration#configure-stateful-reconnect) (<xref:Microsoft.AspNetCore.SignalR.Client.HubConnectionBuilderHttpExtensions.WithStatefulReconnect%2A>) was released with .NET 8 but isn't currently supported for the Azure SignalR Service. For more information, see [Stateful Reconnect Support? (`Azure/azure-signalr` #1878)](https://github.com/Azure/azure-signalr/issues/1878).
 
-* Configure the option in `appsettings.json`:
+:::moniker-end
 
-  ```json
-  "Azure:SignalR:ServerStickyMode": "Required"
-  ```
+We recommend using WebSockets for Blazor apps deployed to Azure App Service. If [WebSockets](https://wikipedia.org/wiki/WebSocket) are disabled, Azure App Service simulates a real-time connection using HTTP Long Polling. HTTP Long Polling is noticeably slower than running with WebSockets enabled, which doesn't use polling to simulate a client-server connection.
 
-> [!NOTE]
-> The following error is thrown by an app that hasn't enabled session affinity:
->
-> > :::no-loc text="blazor.server.js:1 Uncaught (in promise) Error: Invocation canceled due to the underlying connection being closed.":::
+In the event that Long Polling must be used, you may need to configure the maximum poll interval (`MaxPollIntervalInSeconds`, default: 5 seconds, limit: 1-300 seconds), which defines the maximum poll interval allowed for Long Polling connections in the Azure SignalR Service if the service ever falls back from WebSockets to Long Polling. If the next poll request doesn't arrive within the maximum poll interval, the service closes the client connection. The service also cleans up connections when the cached waiting-to-write buffer size is greater than 1 MB for improved service performance.
```

Comment:
@BrennanConroy @mikekistler Is this guidance about long polling specific to using Blazor with the Azure SignalR Service, or does it really apply to all SignalR apps when used with the service?

```suggestion
In the event that HTTP Long Polling must be used instead of WebSockets with the Azure SignalR Service, you may need to configure the maximum poll interval (`MaxPollIntervalInSeconds`, default: 5 seconds, limit: 1-300 seconds), which defines the maximum poll interval allowed for Long Polling connections in the Azure SignalR Service if the service ever falls back from WebSockets to Long Polling. If the next poll request doesn't arrive within the maximum poll interval, the service closes the client connection. The service also cleans up connections when the cached waiting-to-write buffer size is greater than 1 MB for improved service performance.
```

Reply:
Sounds generic. You probably should have ASRS folks look this doc over.

Reply:
Let me know if I should try to find out where/who/when this came from.

Reply:
@chenkennt Can you take a look at this guidance on using HTTP long polling with the Azure SignalR Service? I think this text was added in response to [this GitHub issue discussion](https://github.com/dotnet/aspnetcore/issues/42778#issuecomment-1188852129), but I'm not sure if this content is still relevant and if this is the right place for it.

Reply:
The `MaxPollIntervalInSeconds` content is consistent with the [related Azure SignalR Service docs](https://learn.microsoft.com/azure/azure-signalr/signalr-howto-use#maxpollintervalinseconds). However, it looks like the last statement about the cached waiting-to-write buffer size was [removed from the Azure SignalR Service docs](https://github.com/MicrosoftDocs/azure-docs-pr/pull/272885) by @vicancy, so I believe we should remove it here as well.

```suggestion
In the event that Long Polling must be used, you may need to configure the maximum poll interval (`MaxPollIntervalInSeconds`, default: 5 seconds, limit: 1-300 seconds), which defines the maximum poll interval allowed for Long Polling connections in the Azure SignalR Service if the service ever falls back from WebSockets to Long Polling. If the next poll request doesn't arrive within the maximum poll interval, the service closes the client connection.
```